### DISCUSSION TITLE
Add Support For Composite Keys

### DISCUSSION THREAD

**Discussion by candrewlee14** [dineug/erd-editor#262] [Ideas]
Maybe I'm just not seeing this feature, but I'm having trouble finding an option to create a multicolumn composite key.
It would be convenient if you were to be able to highlight multiple rows, right click, and create a composite key. 

The graphic for this could have a small fork with indicators touching the columns in the key, and a key icon at the end of the stem of the fork.

**Comment by Chahk** [comment]
Same for Unique constraints please!  Right now enabling "Unique" for more than one column on a given table will create multiple constraints.
